### PR TITLE
https://issues.redhat.com/browse/ACM-13627 for 2.12

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: release_notes/release_notes.adoc [hunk 1]
```diff
@@ -14,4 +14,6 @@ Learn about the current release.
 * xref:../release_notes/observability_support.adoc#observability-support[Observability support]
 
 If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view Knowledgebase articles, connect with the Support Team, or open a case. You must log in with your credentials.
-You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ]. 
+You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ].
+
+The documentation team links to the earliest supported {ocp} versions that are tested by Quality Engineering, but later versions are supported. See the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies of for {acm}] for more details about the support version strategy.
```
Comment (swopebe) [inline release_notes/release_notes.adoc]: We don't want to point to the humans doing the work here; in a more informal doc we can. 

I would say "The documentation references the earliest ........., unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see (add support matrix first) and (add lifecycle).


This should go in both ACM and MCE release notes introductions.

**Review by swopebe — CHANGES_REQUESTED** [review]

File: release_notes/release_notes.adoc (hunk 1 above)
Comment (dockerymick) [inline release_notes/release_notes.adoc]:
```suggestion
The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies for {acm}].
```

File: release_notes/release_notes.adoc [hunk 2]
```diff
@@ -14,4 +14,6 @@ Learn about the current release.
 * xref:../release_notes/observability_support.adoc#observability-support[Observability support]
 
 If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view Knowledgebase articles, connect with the Support Team, or open a case. You must log in with your credentials.
-You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ]. 
+You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ].
+
+The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies for {acm}].
```
Comment (dockerymick) [inline release_notes/release_notes.adoc]:
```suggestion
The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```

File: clusters/install_upgrade/install_intro.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 The {mce-short} is a software operator that enhances cluster fleet management. The {mce-short} supports{ocp} and Kubernetes cluster lifecycle management across clouds and data centers. 
 
-Access the link:https://access.redhat.com/articles/7073030[Support matrix] to learn about hub cluster and managed cluster requirements and support.
+The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073030[Support matrix] and the link:https://access.redhat.com/support/policy/updates/openshift[{ocp} Lifecycle policy].
```
Comment (swopebe) [inline clusters/install_upgrade/install_intro.adoc]:
```suggestion
The documentation references the earliest supported {ocp-short} version, unless a specific component or function is introduced and tested only on a more recent version of {ocp-short}.

For full support information, see the link:https://access.redhat.com/articles/7073030[Support matrix]. For life cycle information, see link:https://access.redhat.com/support/policy/updates/openshift[{ocp} Life Cycle policy].
```

File: clusters/release_notes/release_notes_intro.adoc [hunk 4]
```diff
@@ -12,4 +12,6 @@ Learn about the current release.
 
 If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view _Knowledgebase_ articles, connect with the Support Team, or open a case. You must log in with your credentials.
 
-You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ]. 
+You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ].
+
+The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073030[Support matrix] and the link:https://access.redhat.com/support/policy/updates/openshift[{ocp} Lifecycle policy].
```
Comment (dockerymick) [inline clusters/release_notes/release_notes_intro.adoc]:
```suggestion
The documentation references the earliest supported {ocp-short} version, unless a specific component or function is introduced and tested only on a more recent version of {ocp-short}.

For full support information, see the link:https://access.redhat.com/articles/7073030[Support matrix]. For lifecycle information, see link:https://access.redhat.com/support/policy/updates/openshift[{ocp} Life Cycle policy].
```

File: install/install_overview.adoc [hunk 5]
```diff
@@ -9,7 +9,7 @@ Before you install, review the required hardware and system configuration for ea
 
 FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`, then the `multiclusterhub-operator` provides a default list of ciphers. If you upgrade and want FIPS compliance, remove the following two ciphers from the `multiclusterhub` resource: `ECDHE-ECDSA-CHACHA20-POLY1305` and `ECDHE-RSA-CHACHA20-POLY1305`.
 
-Access the link:https://access.redhat.com/articles/7073065[Support matrix] to learn about hub cluster and managed cluster requirements and support.
+The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```
Comment (dockerymick) [inline install/install_overview.adoc]:
```suggestion
The documentation references the earliest supported {ocp-short} version, unless a specific component or function is introduced and tested only on a more recent version of {ocp-short}.

For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```

File: release_notes/release_notes.adoc [hunk 6]
```diff
@@ -14,4 +14,6 @@ Learn about the current release.
 * xref:../release_notes/observability_support.adoc#observability-support[Observability support]
 
 If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view Knowledgebase articles, connect with the Support Team, or open a case. You must log in with your credentials.
-You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ]. 
+You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ].
+
+The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```
Comment (dockerymick) [inline release_notes/release_notes.adoc]:
```suggestion
The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}.

 For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```

**Review by swopebe — APPROVED** [review]

**Pull request merged**